hack/verify-description.sh

```diff
@@ -19,8 +19,12 @@ set -o nounset
 set -o pipefail
 
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${KUBE_ROOT}/hack/lib/init.sh"
 
-cd "${KUBE_ROOT}"
+kube::golang::setup_env
+"${KUBE_ROOT}/hack/build-go.sh" cmd/genswaggertypedocs
```

Comment:
Could you divide verify-description.sh to a hack/verify-description.sh and hack/after-build/verify-description.sh? The after-build/ one shouldn't call build-go.sh. You can take verify-generated-docs.sh as an example. See https://github.com/kubernetes/kubernetes/pull/11841. Thanks.